src/containers/AdminCampaignStats/DeliverabilityStats.tsx

```diff
@@ -88,28 +75,35 @@ const DeliverabilityStats = (props: {
 
   const total = deliveredCount + sendingCount + sentCount + errorCount;
 
+  const highErrorPercent = 25;
+  const campaignErrorPercent = asPercent(errorCount, total);
+  const errorHighlight =
+    campaignErrorPercent !== undefined &&
```

Comment:
Question: In what scenario would this be undefined?

Reply:
Good question, since we default to returning 0 in the function if we pass invalid arguments, so I'll remove the extra check here!